Makefile

```
@@ -57,9 +57,17 @@ BASE ?= calico/base
 BASE_IMAGE ?= $(BASE):latest
 BASE_ARCH_IMAGE ?= $(BASE_IMAGE)-$(ARCH)
 
+
 QEMU ?= calico/qemu-user-static
 QEMU_IMAGE ?= $(QEMU):latest
 
+# Base-image we'll use to build calico/base.
+UBIBASE ?= registry.access.redhat.com/ubi8/ubi-minimal:latest
```

```suggestion
BASE_BASE_IMAGE ?= registry.access.redhat.com/ubi8/ubi-minimal:latest
```
Bit of a stutter, but would align with the args below?